ui/app/AppLayouts/CommunitiesPortal/views/CommunitiesGridView.qml

```diff
@@ -13,7 +13,7 @@ import SortFilterProxyModel 0.2
 StatusScrollView {
     id: root
 
-    property var locale
+    property var locale: Qt.locale()
```

Comment:
Do we need this here?

Reply:
The `StatusCommunityCard` component has a default locale of `Qt.locale("en")`. Not sure why that is, but if it's OK I can change it to `Qt.locale()`, which will allow us to remove the parameter in `CommunitiesGridView`

Reply:
Yes please, `Qt.locale("en")` makes no sense